src/storage/src/compaction/task.rs

```diff
@@ -148,6 +148,7 @@ impl<S: LogStore> CompactionTask for CompactionTaskImpl<S> {
             e
         })?;
         compacted.extend(self.expired_ssts.iter().map(FileHandle::meta));
+        info!("Compacting SST files, input: {compacted:?}, output: {output:?}");
```

Comment:
These logs are annoying.

Reply:
> These logs are annoying.

I intentionally added these logs so that we can track the lifecycle of SST files during troubleshooting. Observability is more important than log efficiency in this period I think.